calculator.js

```diff
@@ -1,2 +1,32 @@
 var calculator = {
-}
+  add: function(x,y) {
+    console.log(x+y);
+    return x+y;
+  },
+  subtract: function(x,y) {
+    console.log(x-y);
+    return x-y;
+  },
+  multiply: function(x,y) {
+    console.log(x*y);
+    return x*y;
+  },
+  divide: function(x,y) {
+    console.log(x/y);
+    return x/y;
+  }
+  exponent: Math.pow(x,y) {
+    console.log(x^,y^);
+    return x^y^;
+  },
```

Comment:
the code above will error out: always test your code! you want to return Math.pow(x,y) like you have above. don't guess--always test!